sdk/storage/Azure.Storage.Blobs/tests/Azure.Storage.Blobs.Tests.csproj

```diff
@@ -30,4 +30,7 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory> 
     </Content> 
   </ItemGroup>
+    <ItemGroup>
+      <Compile Remove="C:\Repos\azure-sdk-for-net\sdk\storage\Azure.Storage.Common\tests\Shared\TamperTransactionalHashPolicy.cs" />
```

Comment:
What is this?  Also I think this is a local path.

Reply:
Must have been generated by Visual Studio at some point. Removing it appears to do nothing.